### PR TITLE
Update game-of-life example

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: game-of-life/index.html [hunk 1]
```diff
@@ -56,21 +56,23 @@ <h1>
...
-var memory = new WebAssembly.Memory({ initial: ((byteSize + 0xffff) & ~0xffff) >>> 16 });
+var memory = new WebAssembly.Memory({
+  initial: ((byteSize + 0xffff) & ~0xffff) >>> 16
+});
 
 // Fetch and instantiate the module
-fetch("build/optimized.wasm")
+fetch("build/release.wasm")
 .then(response => response.arrayBuffer())
 .then(buffer => WebAssembly.instantiate(buffer, {
```
Comment (CountBleck) [inline game-of-life/index.html:66]: Shouldn't you use `WebAssembly.instantiateStreaming()`?

Comment (MaxGraey) [reply game-of-life/index.html:66]: `instantiateStreaming` doesn't support on Safari yet. Or support only in the latest WebKit. So for simplicity better to use non-streamed version

Comment (CountBleck) [reply game-of-life/index.html:66]: In that case, the `interference` example also uses `instantiateStreaming()`. Shouldn't that be changed then?

Comment (MaxGraey) [reply game-of-life/index.html:66]: yes, I think it's worth it

**Pull request merged**